cpu/esp32/Makefile.include

```diff
@@ -51,6 +51,14 @@ ifneq (,$(filter esp_wifi,$(USEMODULE)))
     USEMODULE += esp_wifi_any
 endif
 
+ifneq (, $(filter spiffs, $(USEMODULE)))
+    export RIOT_TEST_TIMEOUT = 300
+endif
+
+ifneq (, $(filter littlefs, $(USEMODULE)))
```

Comment:
For me this is not needed. The test succeeds fast enough master.

Reply:
For `pkg_littlefs`, sometimes I get the time out, sometimes not. It default time seems to be too tight. So I would increase for `pkg_littlefs` too to be sure that it works.